query/src/test/scala/filodb/query/exec/rangefn/AggrOverTimeFunctionsSpec.scala

```diff
@@ -516,32 +607,65 @@ class AggrOverTimeFunctionsSpec extends RawDataWindowingSpec {
         startTime += step * pubFreq
         endTime += step * pubFreq
       }
-      aggregated2 shouldEqual(res)
+      aggregated2 shouldEqual (res)
     }
   }
 
   it("it should correctly calculate zscore") {
-    val data = Seq(15900.0, 15920.0, 15940.0, 15960.0, 15980.0, 16000.0, 16020.0)
-    val data2 = Seq(-15900.0, -15920.0, -15940.0, -15960.0, -15980.0, -16000.0)
-    val data3 = Seq(15900.0, 15920.0, 15940.0, 15960.0, 15980.0, 16000.0, Double.NaN)
-    val params = Seq(StaticFuncArgs(0.01, RangeParams(100,20,500)))
-
-    var rv = timeValueRV(data)
-    val chunkedIt = new ChunkedWindowIteratorD(rv, 160000, 100000, 180000, 100000,
-      new ZScoreChunkedFunctionD(), queryConfig)
-    val aggregated = chunkedIt.map(_.getDouble(1)).toBuffer
-    aggregated(0) shouldBe 1.5
-
-    var rv2 = timeValueRV(data2)
-    val chunkedIt2 = new ChunkedWindowIteratorD(rv2, 160000, 100000, 180000, 100000,
-      new ZScoreChunkedFunctionD(), queryConfig)
-    val aggregated2 = chunkedIt2.map(_.getDouble(1)).toBuffer
-    aggregated2(0) shouldBe (-1.463850109429032 +- 0.0000000001)
+    val data = (1 to 500).map(_.toDouble)
+    val rv = timeValueRV(data)
 
-    var rv3 = timeValueRV(data3)
-    val chunkedIt3 = new ChunkedWindowIteratorD(rv3, 160000, 100000, 180000, 100000,
-      new ZScoreChunkedFunctionD(), queryConfig)
-    val aggregated3 = chunkedIt3.map(_.getDouble(1)).toBuffer
-    aggregated3(0).isNaN shouldBe true
+    (0 until numIterations).foreach { x =>
+      val windowSize = rand.nextInt(100) + 10
+      val step = rand.nextInt(50) + 5
+      info(s"  iteration $x  windowSize=$windowSize step=$step")
```

Comment:
Extra space

Reply:
Actually this has come from develop code : https://github.com/filodb/FiloDB/blob/606a2f301a9265d893e76ac031ec506fe3f8f75a/query/src/test/scala/filodb/query/exec/rangefn/AggrOverTimeFunctionsSpec.scala#L378

I have fixed everywhere though.